### PR TITLE
Update add-storage-space.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a note that each add-on is for 1 GB.

Fixes#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/10502

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/commerce/add-storage-space.md [hunk 1]
```diff
@@ -69,6 +69,9 @@ If you haven't yet bought extra storage for your subscription, you can do that.
 6. On the **How does this look?** page, verify the number of gigabytes of storage you selected, review the pricing information, and then select **Next**.
 7. On the **Complete order** page, verify the total. If you need to make any changes, select **Edit order**. If the order requires a credit check, select the check box. When you're finished, select **Place order** \> **Go to Admin Home**.
 
+> [!NOTE]
+> Each Microsoft Office 365 Extra File Storage Add-on has 1 GB capacity.
```
Comment (JohanFreelancer9) [inline microsoft-365/commerce/add-storage-space.md]:
```suggestion
> Each Microsoft Office 365 Extra File Storage add-on has a 1 GB capacity.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@cmcatee-MSFT Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@cmcatee-MSFT Gentle reminder.

**Review by cmcatee-MSFT — APPROVED** [review]

**Pull request merged**